backend/WebApplication1/Models/FaceAPI/FaceApiError.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace WebApplication1.Models.FaceAPI
+{
+    public class FaceAPIError
```

Comment:
There's no need to name it FaceAPIError if it's in folder Models/FaceAPI, just name it Error